syntax="proto2";

package peleus.plugins.configure;

message http_conn_conf {
    required string name = 1;
    required string ip_port = 2;
    optional string connect_type = 3 [default = "pooled"];
    optional int32 timeout_ms = 4 [default = 150];
    optional int32 max_retry = 5 [default = 3];
    optional int32 backup_request_ms = 6 [default = 150];
};

